src/Microsoft.AspNet.Mvc.Core/DefaultControllerFactory.cs

```diff
@@ -51,5 +74,93 @@ public void ReleaseController(object controller)
                 disposableController.Dispose();
             }
```

Comment:
This seems like it should be part of the activator not the factory. If you created it you might need to clean it up.

Also, I still don't get why the names are the way they are. The factory has `ActivateProperties` and the activator has `Create`. Just seems intentionally obtuse